### PR TITLE
Make setup instructions work from scratch

### PR DESCRIPTION
**Pull request by joakimk**
And allow custom postgres settings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,12 +20,16 @@ You are expected to have Elixir and Postgres installed.
 
     mix deps.get
     mix ecto.setup
-    mix phoenix.server
+    npm install
```
Comment (henrik) [inline README.md:27]: Hm, maybe we should change line 19 to "Phoenix, NPM and Postgres"? I think this bit is part of the Phoenix installation instructions. If so, maybe it doesn't fit in the instructions for this app.

File: config/dev.exs [hunk 2]
```diff
@@ -35,8 +35,8 @@ config :phoenix, :stacktrace_depth, 20
 # Configure your database
 config :toolbox, Toolbox.Repo,
   adapter: Ecto.Adapters.Postgres,
-  username: "postgres",
-  password: "postgres",
+  username: System.get_env("POSTGRES_USER") || "postgres",
+  password: System.get_env("POSTGRES_PASSWORD") || "postgres",
```
Comment (henrik) [inline config/dev.exs:39]: Maybe mention how to set these in the instructions, too?

File: README.md (hunk 1 above)
Comment (henrik) [inline README.md:27]: …but we could link to those instructions.

If it's a super common thing to install Phoenix without NPM, it _could_ make sense to repeat the instructions here for convenience. Then I'd put them first, before any app-specific stuff (like getting deps).

Comment (henrik) [inline README.md:27]: Oh, sorry, I guess "npm install" installs any npm deps. So this does make sense here.

I don't remember having to run this myself, but I guess it ran automatically when the app was generated.

**Comment by joakimk** [comment]
Added docs for the envs now. Good to merge?

File: README.md (hunk 1 above)
Comment (joakimk) [inline README.md:27]: That's probably the case.

**Comment by henrik** [comment]
Yup, thank you! :)

**Pull request merged**